conf/st2.dev.conf

```diff
@@ -11,6 +11,8 @@ mask_secrets = False
 
 [sensorcontainer]
 logging = st2reactor/conf/logging.sensorcontainer.conf
+sensor_node_name = sensornode1
+partition_provider = name:default
```

Comment:
should we call it "all"? Default sounds more like a property which can be assigned to any provider.